assets/js/client.js

```diff
@@ -39,7 +39,15 @@ const link = new ApolloLink.split(
 // Create the Apollo Client instance.
 const client = new ApolloClient({
   link: link,
-  cache: new InMemoryCache(),
+  cache: new InMemoryCache({
+    typePolicies: {
+      FileSet: {
+        fields: {
+          metadata: { merge: false },
+        },
+      },
+    },
+  }),
```

Comment:
Why is this needed?

Reply:
I was getting a warning about updating the cache because FileSetMetadata did not have id's.  

So followed this doc:
https://www.apollographql.com/docs/react/caching/cache-field-behavior/#merging-non-normalized-objects